src/pages/documentation/mobile-core/release-notes.md

```diff
@@ -1,5 +1,17 @@
 # Release Notes
 
+## March 31, 2023
+
+### Android Mobile Core 2.1.1
+
+* Fixed crash during Core boot when migrating from v4 Mobile SDK.
+* Update POM with Apache 2 license URL.
```

Comment:
should we use past tense for all + some rewording -> Updated POM file with the Apache 2 license URL.